src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -389,7 +390,19 @@ class DropdownAutoCompleteMenu extends React.Component {
                       <InputLoadingWrapper>
                         {busy && <LoadingIndicator size={16} mini />}
                       </InputLoadingWrapper>
-                    </Flex>
+                      {menuProps &&
+                        menuProps.showSettingsLink && (
+                          <div>
```

Comment:
why div?

Reply:
If I remove `div` the button becomes the flex-child, and thus full height of the container. We could use `Box` instead, but that doesn't seem worth the import.